CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [7.0.0] - 2022-08-31
 
+### Added
+- Added the possibility to set the alpha channel while loading a model in the meshcat visualizer (https://github.com/robotology/idyntree/pull/1033).
```

Comment:
Can you move it under "Unreleased" ? We already released 7.0.0

Reply:
Ops, my fault 😢 . Done in https://github.com/robotology/idyntree/pull/1033/commits/95d955527dd2a6fa57ab49de3d8e7322d1bb41f9